common/copr_common/background_worker.py

```diff
@@ -14,6 +14,15 @@
 from copr_common.redis_helpers import get_redis_connection
 
 
+@contextlib.contextmanager
+def nullcontext():
+    """
+    contextlib.nullcontext is not available in Python 3.6, but we are still
+    Python 3.6+ compatible because of EL 8
```

Comment:
As per our Slack discussion, this is needed for running Resalloc's agent spawner daemon in RHEL 8 OpenShift containers.  It would be nice to mention the reasons somewhere (at least the commit message).